server/model/taggedBailRemoveModel.ts

```diff
@@ -0,0 +1,67 @@
+import { PrisonApiOffenderSentenceAndOffences, PrisonApiPrisoner } from '../@types/prisonApi/prisonClientTypes'
+import { Adjustment } from '../@types/adjustments/adjustmentsTypes'
+import { AdjustmentType } from './adjustmentTypes'
+import { dateToString } from '../utils/utils'
+
+export default class TaggedBailRemoveModel {
+  constructor(
+    public prisonerDetail: PrisonApiPrisoner,
+    public adjustment: Adjustment,
+    public adjustmentType: AdjustmentType,
+    public sentenceAndOffense: PrisonApiOffenderSentenceAndOffences,
```

Comment:
minor spelling mistake should be `sentenceAndOffence`

Reply:
I have corrected the spelling for this 👍